news-bundle/contao/modules/ModuleNewsReader.php

```diff
@@ -164,6 +166,14 @@ protected function compile()
 			}
 		}
 
+		// Update json ld searchIndexer setting
+		$pageSchema = $responseContext->get(JsonLdManager::class)->getGraphForSchema(JsonLdManager::SCHEMA_CONTAO)->get(ContaoPageSchema::class);
+
+		if (!empty($objArticle->searchIndexer) && 'use_reader_page_setting' !== $objArticle->searchIndexer)
```

Comment:
If you use an empty value for the default, this could just be shortened to

```suggestion
		if ($objArticle->searchIndexer)
```

Reply:
Do you mean changing the DCA from this:

```PHP
'searchIndexer' => array
(
	'filter'                  => true,
	'inputType'               => 'select',
	'options'                 => array('use_reader_page_setting', 'always_index', 'never_index'),
	'eval'                    => array('maxlength'=>32, 'tl_class'=>'w50'),
	'reference'               => &$GLOBALS['TL_LANG']['tl_news'],
	'sql'                     => "varchar(32) NOT NULL default ''"
),
```

To something like this?

```PHP
'searchIndexer' => array
(
	'filter'                  => true,
	'inputType'               => 'select',
	'options'                 => array('' => 'use_reader_page_setting_label', 'always_index' => 'always_index_label', 'never_index' => 'never_index_label'),
	'eval'                    => array('maxlength'=>32, 'tl_class'=>'w50'),
	'reference'               => &$GLOBALS['TL_LANG']['tl_news'],
	'sql'                     => "varchar(32) NOT NULL default ''"
),
```

To always have an empty value as default?

Reply:
No, something like this:

```php
'searchIndexer' => array
(
	'filter'                  => true,
	'inputType'               => 'select',
	'options'                 => array('always_index', 'never_index'),
	'eval'                    => array('maxlength'=>32, 'includeBlankOption'=>true, 'blankOptionLabel'=>&$GLOBALS['TL_LANG']['MSC']['searchIndexerBlankOption'], 'tl_class'=>'w50'),
	'reference'               => &$GLOBALS['TL_LANG']['MSC']['searchIndexerOptions'],
	'sql'                     => "varchar(32) NOT NULL default ''"
),
```

`MSC.searchIndexerBlankOption` would then say `Use robots tag`.

Reply:
I'll try it...